app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/initialgainorloss/initial_gain_or_loss.scala.html

```diff
@@ -92,28 +106,21 @@
 }
 
 
-
-@title = @{messages(s"$key.title")}
-
-@mainTemplate(title = title, userType = request.userType) {
-@backLinkComponent(backLink)
-@caption(messages("initialGainOrLoss.caption"))
-@if(hasErrors) {
-    @errorSummary(form)
+@title = @{
+    messages(s"$key.title$userKey")
```

Comment:
the convention we've used everywhere else is `key.userKey.x` - it'd be more consistent to go with that approach